newsroom/types/products.py

```diff
@@ -7,3 +14,33 @@ class ProductType(str, Enum):
     WIRE = "wire"
     AGENDA = "agenda"
     NEWS_API = "news_api"
+
+
+class ProductResourceModel(NewshubResourceModel):
+    name: Annotated[
+        str,
+        validate_multi_field_iunique_value_async("products", ["name", "product_type"]),
+    ]
+    description: str | None = None
+    sd_product_id: str | None = None
+    query: str | None = None
+    planning_item_query: str | None = None
+    is_enabled: bool = True
+    product_type: ProductType = ProductType.WIRE
+    navigations: list[
+        Annotated[
+            ObjectId,
+            validators.validate_data_relation_async("navigations"),
+            validate_valid_objectid("navigations"),
+        ]
+    ] = Field(default_factory=list)
+
+    # obsolete
```

Comment:
@eos87 Originally product permissions where on the Product itself. This was changed so these permissions are now stored on Company and Use resources